ospfd/ospfd.c

```diff
@@ -248,6 +248,9 @@ static struct ospf *ospf_new(unsigned short instance, const char *name)
 	} else {
 		new->vrf_id = VRF_DEFAULT;
 		vrf = vrf_lookup_by_id(VRF_DEFAULT);
+
+		assert(vrf);
```

Comment:
assert will completely stop ospfd process in case of error. Few lines above, (in the if part) code check if vrf is not null before calling `ospf_vrf_link()`
I would prefer such test instead of assert.

Reply:
Thank you, I missed that. BTW, because of ospf_vrf_link() is called at both 'if' and 'else', I'll put the ospf_vrt_link() call out of the condition.